xml/admin_software.xml

```diff
@@ -279,6 +279,15 @@
 
   <sect2 xml:id="sec.admin.software.transactional-updates.installation">
    <title>Applying Updates</title>
+   <note>
+    <title>Registering nodes</title>
+    <para>
+     In order to update nodes, you should first register them at SUSE Customer 
+     Center, SMT or RMT. If you did not register during the installation, see 
+     <link xlink:href="https://www.suse.com/documentation/suse-caasp-3/singlehtml/book_caasp_deployment/book_caasp_deployment.html#sec.configuration.suseconnect"/>
+     &mdash; the SUSE CaaS Platform 3 Deployment Guide.
```

Comment:
Please replace the product names in this paragraph with the respective entities. For example instead of writing out `SUSE Customer Center` you should use `&scc;`. The entities are defined here: 

https://github.com/SUSE/doc-caasp/blob/maintenance/CaaS3/xml/entity-decl.ent#L25

most importantly instead of having to write out all of `SUSE CaaS Platform 3` just use `&productname; &productnumber;`. This also makes it a lot easier to maintain versions because you just need to change the version number in the entities file and not across X locations around all files.

Please check out the Entities section in the Hitchhikers Guide (http://docserv.nue.suse.com/documents/Style_Guide/docu_styleguide/html/#sec.entity) and look through the entities file to get familiar with what you can replace.